CHANGELOG.md

```diff
@@ -3,7 +3,14 @@
 
 ### BREAKING CHANGES
 
-* *withDisableReportOnExit* method has been removed from Options. Clients should instead call *AbstractTracer.flush()* on exit. 
+* *withDisableReportOnExit* method has been removed from Options. Clients should instead call *AbstractTracer.flush()* on exit.
```

Comment:
Not super important, but it's not only that that option was removed, it's also a change in the effective default behavior (regardless of whether that option exists). Previously, the Tracer automatically flushed on exit. Now it _never_ automatically flushes on exit. (Right?)